### PR TITLE
UI Done using  Quasar

### PR DESCRIPTION
**Pull request by sanchitno1**
Screenshot (73)
Screenshot (74)

### CODE REVIEW

File: src/components/signup.vue [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      }
+      const data={      //this code demonstrates that our data is reflected in console properly.
+        name: this.name,
+        last_name:this.last_name,
+        age: this.age,
+        phone:this.phone,
+        email:this.email,
+        password:this.password,
+      };
+      console.log(data)
```
Comment (priyankashrma) [inline src/components/signup.vue]: Remove console log, Before raising PR make sure you remove all console logs.

File: src/components/signup.vue [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      <q-input
+        rounded filled
+        v-model="name"
+        label="First name *"
+        lazy-rules
+        :rules="[ val => val && val.length > 0 || 'Please type your Valid First Name']"
+      ></q-input>
+      <q-input
+        rounded filled
+        v-model="last_name"
```
Comment (santhoshgl) [inline src/components/signup.vue]: @sanchitno1 always camelCase please change from last_name to lastName cc @priyankashrma

File: src/components/signup.vue [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      @submit="onSubmit"
+      @reset="onReset"
+      class="q-gutter-md"
+    >
+      <q-input
+        rounded filled
+        v-model="name"
+        label="First name *"
+        lazy-rules
+        :rules="[ val => val && val.length > 0 || 'Please type your Valid First Name']"
```
Comment (santhoshgl) [inline src/components/signup.vue]: @sanchitno1 keep it simple **Please enter your first name**

File: src/components/signup.vue [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+      ></q-input>
+      <q-input
+        rounded filled
+        v-model="last_name"
+        label="Last name *"
+        lazy-rules
+        :rules="[ val => val && val.length > 0 || 'Please type your Valid Last Name']"
```
Comment (santhoshgl) [inline src/components/signup.vue]: @sanchitno1 keep it simple **Please enter your last name**

File: src/components/signup.vue [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+      ></q-input>
+
+      <q-input
+        rounded filled
+        type="number"
+        v-model="age"
+        label="Your age *"
+        lazy-rules
+        :rules="[
+          val => val !== null && val !== '' || 'Please type your age',
```
Comment (santhoshgl) [inline src/components/signup.vue]: @sanchitno1 keep it simple **Please enter your age**

File: src/components/signup.vue [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+          val => val > 0 && val.length < 11 || 'Please type a correct number'
+        ]"
+      ></q-input>
+      <q-input
+        rounded filled
+        v-model="email"
+        label="Enter your email-id *"
+        lazy-rules
+        :rules="[
+           val => val && val.length > 6 || 'Please type your Valid User Name']"
```
Comment (santhoshgl) [inline src/components/signup.vue]: @sanchitno1 keep it simple **Please enter your user name**

File: src/components/signup.vue [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+</template>
+
+
+<script>
+export default {
+  name:'signup',
+  data () {
+    return {
+      name: null,
+      last_name:null,
```
Comment (santhoshgl) [inline src/components/signup.vue]: change it lastName

File: src/components/signup.vue [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+      if (this.accept !== true) {
+           alert('You need to accept the license and terms first')
+      }
+      else {
+          this.$router.push('/submitted')
+       
+      }
+      const data={      //this code demonstrates that our data is reflected in console properly.
+        name: this.name,
+        last_name:this.last_name,
```
Comment (santhoshgl) [inline src/components/signup.vue]: change it to lastName

File: src/components/signup.vue [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+        email:this.email,
+        password:this.password,
+      };
+      console.log(data) 
+    },
+
+    onReset () {
+      this.name = null
+      this.age = null
+      this.last_name = null
```
Comment (santhoshgl) [inline src/components/signup.vue]: change it to lastName

**Pull request merged**